### PR TITLE
Add ConductR specifics

### PR DESCRIPTION
**Pull request by huntc**
Hopefully self-explanatory. Tested in relation to the ConductR/Lagom video be prepared.

### CODE REVIEW

**Comment by markusjura** [comment]
LGTM

File: front-end/bundle-configuration/default/runtime-config.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+export APPLICATION_SECRET=PPjOW0n2aV?s@6RdiNV@7/5xJhiiKTzk[VdHjkU9YHit8sLHoJ1rp0DCCn6b=lXt
```
Comment (dotta) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: Shouldn't this be changed to `changeme` (or similar)?

Comment (markusjura) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: This is the application secret for the prod mode. In the `application.conf` the secret is "changeme". This shell script will override the application secret in ConductR.

Comment (huntc) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: I don't think so. We are forcing users to think about alternate config they must perform a deliberate action to load it - and if they don't load the config then changeme will be there. Make sense?

Comment (dotta) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: Alright, you guys know better, but I feel a comment about what this file is for and how it's intended to be used would be useful.

Comment (markusjura) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: Yes, we certainly can add a comment. Note that we need to update the project anyway once the corresponding ConductR video is hosted on lightbend.com.

Comment (dotta) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: Ok. So, do you want to add a comment or shall I merge?

Comment (markusjura) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: Let me create an issue for that. We'll add the comment once the video has been hosted.

Comment (dotta) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: Sounds good, I'll merge right away.

Comment (markusjura) [inline front-end/bundle-configuration/default/runtime-config.sh:2]: Issue: https://github.com/lagom/activator-lagom-java-chirper/issues/4

**Pull request merged**